lib/experimental/class-wp-theme-json-resolver-gutenberg.php

```diff
@@ -34,15 +34,20 @@ public static function get_theme_data( $deprecated = array() ) {
 		if ( null === static::$theme ) {
 			$theme_json_data = static::read_json_file( static::get_file_path_from_theme( 'theme.json' ) );
 			$theme_json_data = static::translate( $theme_json_data, wp_get_theme()->get( 'TextDomain' ) );
+
+			$original_theme_json = new WP_Theme_JSON_Gutenberg( $theme_json_data );
+			gutenberg_register_webfonts_from_theme_json( $original_theme_json->get_settings() );
+
 			$theme_json_data = gutenberg_add_registered_webfonts_to_theme_json( $theme_json_data );
 			static::$theme   = new WP_Theme_JSON_Gutenberg( $theme_json_data );
 
 			if ( wp_get_theme()->parent() ) {
 				// Get parent theme.json.
 				$parent_theme_json_data = static::read_json_file( static::get_file_path_from_theme( 'theme.json', true ) );
 				$parent_theme_json_data = static::translate( $parent_theme_json_data, wp_get_theme()->parent()->get( 'TextDomain' ) );
-				$parent_theme_json_data = gutenberg_add_registered_webfonts_to_theme_json( $parent_theme_json_data );
-				$parent_theme           = new WP_Theme_JSON_Gutenberg( $parent_theme_json_data );
+
+				$parent_theme = new WP_Theme_JSON_Gutenberg( $parent_theme_json_data );
+				gutenberg_register_webfonts_from_theme_json( $parent_theme->get_settings() );
```

Comment:
I think that we should be calling `gutenberg_add_registered_webfonts_to_theme_json` on the parent theme as well?